common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -48,9 +48,9 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-contributions-global-mobile-banner-design",
```

Comment:
I've removed the mobile-banner-design test, which is no longer running

Reply:
thank you